Generative process of Author-Topic (#125)
MrLongbottom/P10-Article: sections/preliminaries.tex

\subsection{Author-topic \gls{lda}}\label{subsec:auth_prelim} \vejleder{introducer modellen som den er i \cite{author_topic_2012}, gerne flere detaljer}
\citet{author_topic_2012} present the author-topic model.
It seeks to find topics based on author metadata, and is based on the assumption that authors prefer to write about specific topics.
In this model, there are no document-topic distributions $\theta$, but rather one author-topic distribution for each author.

> The reason for this is to find the interest of authors, within the documents, that we are analyzing.

Reviewer: ```suggestion
The reason for this is to find the interest of authors within the documents that we are analyzing.
```

> The generative process for the author-topic model is similar to that of the \gls{lda} with a few minor changes.

Reviewer: ```suggestion
The generative process for the author-topic model is similar to that of the \gls{lda} model with a few minor changes.
```

> The model assumes that there are multiple authors for each document which is modeled by the $x$ variable in \autoref{fig:original_author_lda}

Reviewer: ```suggestion
The model assumes that there are multiple authors for each document which is modeled by the $\boldsymbol{a_d}$ and $x$ variables in \autoref{fig:original_author_lda}.
```

> Before drawing a word topic, we need to select an author before drawing a topic which is executed in B.a.

Reviewer: ```suggestion
Before drawing a word topic, we need to select an author $x$ from $\boldsymbol{a_d}$.
The generative process is seen in the following steps:
```

> The reason for this is to find the interest of authors, within the documents, that we are analyzing.
> The generative process for the author-topic model is similar to that of the \gls{lda} with a few minor changes.
> The model assumes that there are multiple authors for each document which is modelled by the $x$ variable in \autoref{fig:original_author_lda}
> Before drawing a word topic, we need to select an author before drawing a topic which is executed in B.a.

Author: ```suggestion
The reason for this is to find the interest of authors, within the documents, that we are analyzing.
The generative process for the author-topic model is similar to that of the \gls{lda} with a few minor changes.
The model assumes that there are multiple authors for each document which is modeled by the $x$ variable in \autoref{fig:original_author_lda}
Before drawing a word topic, we need to select an author before drawing a topic which is executed in B.a.
```

\vspace{\topsep}
\begin{enumerate}

> 	\item Draw topic proportion $\theta_a \sim Dirichlet(\alpha)$

Reviewer: ```suggestion
	\item For each author, draw topic proportion $\theta_a \sim Dirichlet(\alpha)$
```

	\item For each word $n$ in the document:
	\begin{enumerate}

> 		\item Draw author assignment $x \sim \boldsymbol{a_d}$

Reviewer: ```suggestion
		\item Draw author assignment $x \sim U(\boldsymbol{a_d})$
```
Ved ikke om man kan skrive det uden et distribution symbol. Mit forslag er at bruge U som uniform distribution symbol, hvis du tænker at den distribution type passer med hvordan de udtrækker x.

> 		\item Draw topic assignment $z_{d,n} \sim Mult(\theta_a)$

Reviewer: ```suggestion
		\item Draw topic assignment $z_{d,n} \sim Mult(\theta_x)$
```
tror jeg vil bruge x her, for at understrege at det er den udtrukne author der bruges

		\item Draw word $w_{d,n} \sim Mult(\beta_{z_{d,n}})$
	\end{enumerate}
\end{enumerate}

> \vspace{\topsep}

Reviewer: ```suggestion
\vspace{\topsep}
Here $U()$ denotes the discrete uniform distribution.
```
Hvis vi vælger at bruge U
